From c8dbc06b05fb739f6a36f98786acf5d60eb301e4 Mon Sep 17 00:00:00 2001
From: Sergey Temerkhanov <s.temerkhanov@gmail.com>
Date: Mon, 16 May 2016 15:32:00 -0400
Subject: [PATCH 0446/1239] thunderx: Remove stale node1 mapping entries from
 config

81xx is not NUMA capable, so it needs only Node 0 addresses

Signed-off-by: Sergey Temerkhanov <s.temerkhanov@gmail.com>
---
 include/configs/thunderx_81xx.h | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/include/configs/thunderx_81xx.h b/include/configs/thunderx_81xx.h
index cf43069e49..826e154f6b 100644
--- a/include/configs/thunderx_81xx.h
+++ b/include/configs/thunderx_81xx.h
@@ -7,8 +7,6 @@
 #ifndef __THUNDERX_81XX_H__
 #define __THUNDERX_81XX_H__
 
-#define DEBUG
-
 #define CONFIG_SPECIAL_SYNC_HANDLER
 
 /*#define CONFIG_ARMV8_SWITCH_TO_EL1*/
@@ -47,21 +45,12 @@
 					 {0x880000000000UL, 0x40000000000UL, \
 					  PTL2_MEMTYPE(MT_DEVICE_NGNRNE) |   \
 					  PTL2_BLOCK_NON_SHARE},	     \
-					 {0x900000000000UL, 0x40000000000UL, \
-					  PTL2_MEMTYPE(MT_DEVICE_NGNRNE) |   \
-					  PTL2_BLOCK_NON_SHARE},	     \
-					 {0x940000000000UL, 0x40000000000UL, \
-					  PTL2_MEMTYPE(MT_DEVICE_NGNRNE) |   \
-					  PTL2_BLOCK_NON_SHARE},	     \
-					 {0x980000000000UL, 0x40000000000UL, \
-					  PTL2_MEMTYPE(MT_DEVICE_NGNRNE) |   \
-					  PTL2_BLOCK_NON_SHARE},	     \
 					 {0xffffffffffffUL, 0xfffffffffffUL, \
 					  PTL2_MEMTYPE(MT_DEVICE_NGNRNE) |   \
 					  PTL2_BLOCK_NON_SHARE},	     \
 					}
 
-#define CONFIG_SYS_MEM_MAP_SIZE		7
+#define CONFIG_SYS_MEM_MAP_SIZE		4
 
 #define CONFIG_COREID_MASK		0xffffff
 #define CONFIG_SYS_FULL_VA
-- 
2.29.0

